src/SpecflowAutomationTest/Calculator.feature

```diff
@@ -1,6 +1,88 @@
-﻿Feature: Calculator
+Feature: Calculator
 	Simple calculator for applying math operators on two numbers
 
 	#Please define and implement ALL needed scenarios for testing a simple calculator.
 	#The calculatoor has only one feature and it is to apply math operations on two numbers.
-	#The calculator is located at SampleLib project and has a method called "Calculate".
\ No newline at end of file
+	#The calculator is located at SampleLib project and has a method called "Calculate".
+
+#basic scenarios
+  
+ Scenario: Add two numbers
+ Given a digit is 30
+ And b digit is 90
+ When Sum them
+ Then the result is 120
+
+  Scenario: Substract two numbers
+  Given a digit is 40
+  And b digit is 20
+  When Substract them
```

Comment:
Not all possible values of the MathOperatorType  enum are tested, "Power" is skipped.